main/res/values/strings.xml

```diff
@@ -1113,6 +1114,7 @@
     <string name="navigation_direct_navigation">Direct Navigation</string>
     <string name="navigation_target">Target</string>
     <string name="err_nav_no_coordinates">Cannot start navigation with no coordinates</string>
+    <string name="osmand_marker_cgeo">Marker from c:geo</string>
```

Comment:
In the marker list in OsmAnd, if there is no name for the coordinates.
![markers in osmand](https://cloud.githubusercontent.com/assets/1462367/17627169/9bb4a6e2-60b0-11e6-948c-f90476d009bb.png)